server/controllers/importers/sif.js

```diff
@@ -122,20 +122,23 @@ module.exports = function (sif) {
                 });
             }
         });
+  }
+
+  var network = {
+    genes: emptySifFile ? [] : genes.map(function (geneName) {
+      return { name: geneName };
+    }),
```

Comment:
Is this the only use for `emptySifFile`? If this is its sole purpose, why can’t `genes` just be left as `[]` if the SIF is empty? An empty list will `map` to an empty list.